src/main/scala/cromwell/binding/values/WdlBoolean.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell.binding.values
+
+import cromwell.binding.WdlExpressionException
+import cromwell.binding.types.WdlBooleanType
+
+case class WdlBoolean(value: Boolean) extends WdlPrimitive {
+  val wdlType = WdlBooleanType
+  override def equals(rhs: WdlValue): WdlBoolean = {
+    rhs match {
+      case r:WdlBoolean => WdlBoolean(value == r.value)
+      case _ => invalid(s"$value || $rhs")
+    }
+  }
+  override def lessThanOrEqual(rhs: WdlValue): WdlBoolean = equals(rhs)
+  override def greaterThanOrEqual(rhs: WdlValue): WdlBoolean = equals(rhs)
```

Comment:
On this one I'd prefer that booleans are just booleans and that things like `1 + false` and `true > false` aren't supported. But if you really want to mimic a real life prog lang i'll fall back to my statement of do whatever python does